tests/Http/SignupTest.php

```diff
@@ -393,7 +393,7 @@ public function testSignupsIndexAsAdminWithFilters()
         ]);
 
         // Create three more signups with different northstar_id, campaign_id, and campaign_run_id
-        $secondNorthstarId = $this->faker->northstar_id;
+        $secondNorthstarId = '5554eac1a59dbf117e8b456z';
```

Comment:
Why do we have to set this one explicitly but not the other one?

Reply:
@katiecrane good q! I have seen on my local that since it is randomized, sometimes they get set to the same string. So the results get borked since `$northstarId` and `$secondNorthstarId` think they're the same user.

Reply:
Ooooh, _tricky!!_ You should also be able to avoid this using Faker's `unique()` provider:

```php
$northstarId = $this->faker->unique()->northstar_id;
//...
$secondNorthstarId = $this->faker->unique()->northstar_id;
```

Reply:
omg that is way better and good to know - thanks @DFurnes ! I'll change now!